index.html

```diff
@@ -4,285 +4,146 @@
-                <li class="flex">
-                    <img class="flex-img" src="image/descktop/JS.svg" alt="">
-                    <p>JavaSript</p>
-                  </li>
-                  <li class="flex">
-                    <img class="flex-img" src="image/descktop/Html.svg" alt="">
-                    <p>HTML</p>
-                  </li>
-                  <li class="flex">
-                    <img class="flex-img" src="image/descktop/Css.svg" alt="">
-                    <p>CSS</p>
-                  </li>
-            </ul>
-            <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
-            <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
-        </ul>
-    </div>
-</div>
-</section>
-<section class="cardcontact">
-    <form class="cardcontact1" action="https://formspree.io/f/mwkzayzq" method="post" >
-        
-        <div class="sectioncontact">
-            <h2  class="contactme" >Contact me </h2>
-            <p class="contact"> If you have an application you are interested 
-            in developing, a feature that you need built or a project that 
-            needs coding. I’d love to help with it  </p>
-        </div>
-        <fieldset class="filedset">
-            <input class="texform" type="text"  id="Name"   classe="form-control" placeholder="Alexandra Loremipsum" minlength="4" maxlength="30" required />
-            <input class="texform"  type="email"  id="email"  name="email"   placeholder="alexandra@gmail.com"  required  />
-           <input class="texform2" name="comment" id="comment" cols="50" rows="6" placeholder="Write your message here..." required >
-        </fieldset>
-            
-         <div class="buttonsee2" id="contactbuttonid">
-            <button class="contactbutton"> Get It Touch </button>
-        </div>
+    <section>
+        <h2>Featured Speakers</h2>
+        <div id="lineid" class="line"></div>
+        <section class="gridcontainer" id="Portfolio">
            
-    </form>
+        </section>
+    </section>
     
-  </section>
+     <section id="partenerid" class="section-partner">
+                <div class="partners">
+                    <h2 class="partner-head">Partener</h2>
+                <div class="line"></div>
+                <div class="parteners-img">
+                    <img class="img-part" src="./image/Partener/ethio telecom.jfif" alt="">
+                    <img class="img-part"  src="./image/Partener/CNET.jfif" alt="">
+                    <img class="img-part" src="./image/Partener/huwawei.jfif" alt="">
+                    <img class="img-part" src="./image/Partener/download.jfif" alt="">
+                </div>
+                </div>
+                <div class="right-reserved">
+                    <img class="ict-logo" src="./image/ICT-Logo-2015.jpg" alt="">
+                    <p>2023 Creative Commons ICT EXPO  some Rights Reserved</p>
+                </div>
+    </section>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to change the `section` tag to a `footer` tag as the previous reviewer suggested for the about page, I highly recommend making this change so your code is the same across all the pages